### DISCUSSION TITLE
[Feature Request] AWS EventBridge Support as an alternative to Webhooks

### DISCUSSION THREAD

**Discussion by lriggle-strib** [directus/directus#2799] [Feature Requests]
Note: I'm not suggesting eliminating webhooks, I'm suggesting customers could have a choice between the two.

AWS EventBridge is an event bus system to receive incoming events and route them to appropriate places (URL/API endpoint, Lambda, etc). Support for this could be an alternative to setting up webhooks within Directus. Control of the data flow within AWS can be as broad or as granular as the project requires (All Creates go to a location, All actions for this collection go to a specific location, only deletes for a certain collection goes to a specific location).

EventBridge offers flexibility that webhooks do not, in that the data sent to EventBridge has more options of where it can end up (Example: Lambda, SQS), rather than the only option being a URL endpoint.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
